docs/Extensions.md

```diff
@@ -37,6 +37,58 @@ You can see the [vscode-xml-maven](https://github.com/angelozerr/vscode-xml-mave
 
 ![demo of vscode xml maven suggesting different scopes for a dependency](./images/vscode-xml-maven.gif)
 
+### Debugging your extension
```

Comment:
I would change with `Debugging your Java LemMinx extension` to avoid confusing with vscode extension which register the LemMinx extension.